db/migrate/20211001151212_update_patient_table_string_lengths.rb

```diff
@@ -0,0 +1,185 @@
+    change_column :patients, :foreign_monitored_address_county, :string, limit: 200
+
+    # Contact
+    change_column :patients, :preferred_contact_method, :string, limit: 32
+    change_column :patients, :preferred_contact_time, :string, limit: 16
+    change_column :patients, :primary_telephone, :string, limit: 16
+    change_column :patients, :primary_telephone_type, :string, limit: 16
+    change_column :patients, :secondary_telephone, :string, limit: 16
+    change_column :patients, :secondary_telephone_type, :string, limit: 16
+    change_column :patients, :international_telephone, :string, limit: 50
+    change_column :patients, :email, :string, limit: 200
+
+    # Arrival
+    change_column :patients, :port_of_origin, :string, limit: 200
+    change_column :patients, :flight_or_vessel_number, :string, limit: 200
+    change_column :patients, :flight_or_vessel_carrier, :string, limit: 200
+    change_column :patients, :port_of_entry_into_usa, :string, limit: 200
+    change_column :patients, :source_of_report, :string, limit: 32
+    change_column :patients, :source_of_report_specify, :string, limit: 200
+
+    # Additional Planned Travel
+    change_column :patients, :additional_planned_travel_type, :string, limit: 16
+    change_column :patients, :additional_planned_travel_destination, :string, limit: 200
+    change_column :patients, :additional_planned_travel_destination_state, :string, limit: 200
+    change_column :patients, :additional_planned_travel_destination_country, :string, limit: 200
+    change_column :patients, :additional_planned_travel_port_of_departure, :string, limit: 200
+
+    # Exposure
+    change_column :patients, :potential_exposure_location, :string, limit: 200
+    change_column :patients, :potential_exposure_country, :string, limit: 200
+    change_column :patients, :contact_of_known_case_id, :string, limit: 200
+    change_column :patients, :was_in_health_care_facility_with_known_cases_facility_name, :string, limit: 200
+    change_column :patients, :laboratory_personnel_facility_name, :string, limit: 200
+    change_column :patients, :healthcare_personnel_facility_name, :string, limit: 200
+    change_column :patients, :member_of_a_common_exposure_cohort_type, :string, limit: 200
+
+    # Monitoring
+    change_column :patients, :exposure_risk_assessment, :string, limit: 32
+    change_column :patients, :monitoring_plan, :string, limit: 64
+    change_column :patients, :case_status, :string, limit: 16
+    change_column :patients, :public_health_action, :string, limit: 64
+    change_column :patients, :monitoring_reason, :string, limit: 64
+    change_column :patients, :follow_up_reason, :string, limit: 64
+
+    # Other
+    change_column :patients, :legacy_primary_language, :string, limit: 64
+    change_column :patients, :legacy_secondary_language, :string, limit: 64
+    change_column :patients, :time_zone, :string, limit: 64
+    change_column :patients, :submission_token, :binary, limit: 64
```

Comment:
Does the limit on this make a difference since it is binary and not string? I thought this was stored in a separate data structure internally.